example.py

```diff
@@ -1123,6 +1204,25 @@ def core(self, slothy):
         slothy.config.split_heuristic = False
         slothy.optimize_loop("layer5678_start")
 
+        slothy.config = Config(self.arch, self.target)
```

Comment:
@dop-amin I think that's the issue -- this overwrites anything that is setup by the parent class, including the dry run parameters.

It's not ideal that one can't do what you did, but at the moment you have to make a copy of the config at the start of `core()`.

Reply:
Thanks for pointing me to this. Should be fixed.